### ISSUE TITLE
@types/react-responsive definition update

### ISSUE THREAD

**Issue by gndelia** [DefinitelyTyped/DefinitelyTyped#37268]
[react-responsive](https://github.com/contra/react-responsive) types are out of date

**Example code**

```jsx
import { Context as ResponsiveContext } from 'react-responsive';
```

and I get 

> @types/react-responsive"' has no exported member 'Context'.

Reviewing react-responsive index.d.ts I see

> // Type definitions for react-responsive 3.0

while the library is in [version 7](https://www.npmjs.com/package/react-responsive) (with an 8-beta version)

It would be ideal to upgrade the types to the latest version 7 (or 8 if it gets released before)

**Comment by gndelia** [comment]
I kind of have the PR, I just need to test it a bit more and then release it. Hopefully next week (I've been busy with university).

On the other thand, there's a new release as well, version 8.0.0 https://www.npmjs.com/package/react-responsive so I will try to add in another PR those new types as well (hooks)

**Comment by gndelia** [comment]
well someone else did it faster than me 😭 

This should be closed, as it was fixed by #38329
